testsuites/CBLTester/CBL_Functional_tests/APITests/test_collection.py

```diff
@@ -0,0 +1,264 @@
+import pytest
+from keywords.utils import random_string, log_info
+
+# These test cases are only for checking the framework
+
+
+@pytest.mark.usefixtures("class_init")
+class TestDatabase(object):
+
+    @pytest.mark.parametrize("db_name", [
+        random_string(1),
+        random_string(6),
+        random_string(128),
+        "_{}".format(random_string(6)),
+        "{}_".format(random_string(6)),
+        "_{}_".format(random_string(6)),
+        random_string(6, digit=True),
+        random_string(6).upper(),
+    ])
+    # Test to check default scope name in CBL.
+    def test_defaultScope(self, db_name):
```

Comment:
Do we need this _default scope name test? Maybe we have that test in the unit tests and that should be sufficient? Since the QE tests are heavy and take long to run, we might be able to save some time here.